### PR TITLE
KAFKA-4079: Documentation for secure quotas

### PR DESCRIPTION
**Pull request by rajinisivaram**
Details in KIP-55.

### CODE REVIEW

**Comment by rajinisivaram** [comment]
@junrao Can you review the docs as well, please? Thank you...

File: docs/ops.html [hunk 1]
```diff
@@ -340,23 +340,71 @@
...
+Configure custom quota for client-id:
 <pre>
-> bin/kafka-configs.sh  --zookeeper localhost:2181 --alter --add-config 'producer_byte_rate=1024,consumer_byte_rate=2048' --entity-name clientA --entity-type clients
-Updated config for clientId: "clientA".
+> bin/kafka-configs.sh  --zookeeper localhost:2181 --alter --add-config 'producer_byte_rate=1024,consumer_byte_rate=2048' --entity-type clients --entity-default
+Updated config for entity: default client-id.
 </pre>
 
-Here's how to describe the quota for a given client.
+Here's how to describe the quota for a given (user, client-id):
```
Comment (junrao) [inline docs/ops.html:386]: Would it be useful to add an example that we can describe all configs for a given entity-type w/o specifying the entity-name?

Comment (rajinisivaram) [reply docs/ops.html:386]: @junrao Thank you for the review. Have added describe all.

**Comment by junrao** [comment]
@rajinisivaram : Thanks for the patch. LGTM

**Pull request closed without merging**